### PR TITLE
Add information and playbooks for katello 3.6

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: config/versions.yaml [hunk 1]
```diff
@@ -16,6 +16,15 @@ installers:
       - 'debian9'
       - 'ubuntu1604'
 
+  - foreman: '1.17'
+    katello: '3.6'
+    puppet: 5
+    boxes:
+      - 'centos7'
+      - 'debian8'
```
Comment (evgeni) [inline config/versions.yaml]: I *think* foreman 1.17 does not support Debian 8 anymore, but @mmoll can confirm that

**Comment by evgeni** [comment]
LGTM besides the Debian 8 comment.

File: config/versions.yaml (hunk 1 above)
Comment (ekohl) [reply config/versions.yaml]: It was indeed dropped.

File: pipelines/pipeline_katello_35_to_36_upgrade.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      - "--disable-system-checks"
+      - "--foreman-proxy-trusted-hosts {{ server_fqdn.stdout }}"
+      - "--foreman-proxy-trusted-hosts {{ ansible_nodename }}"
+      - "--foreman-proxy-foreman-base-url https://{{ server_fqdn.stdout }}"
+      - "--foreman-proxy-register-in-foreman true"
+      - "--foreman-proxy-oauth-consumer-key {{ oauth_consumer_key.stdout }}"
+      - "--foreman-proxy-oauth-consumer-secret {{ oauth_consumer_secret.stdout }}"
+      - "--foreman-proxy-content-certs-tar {{ foreman_proxy_content_certs_tar }}"
+      - "--foreman-proxy-content-parent-fqdn {{ server_fqdn.stdout }}"
+      - "--foreman-proxy-content-pulp-oauth-secret {{ pulp_oauth_secret.stdout }}"
```
Comment (ekohl) [inline pipelines/pipeline_katello_35_to_36_upgrade.yml]: This line should be dropped similar to https://github.com/theforeman/forklift/pull/594

File: pipelines/pipeline_katello_36.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      - foreman-installer-katello
+  roles:
+    - foreman_proxy_content
+    - foreman_installer
+
+- hosts: pipeline-katello-3.6-centos7
+  become: true
+  vars:
+    bats_tests:
+      - fb-test-katello.bats
```
Comment (ehelms) [inline pipelines/pipeline_katello_36.yml]: I think we may want to add the `fb-test-puppet.bats` here

Comment (zjhuntin) [reply pipelines/pipeline_katello_36.yml]: Added

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**